cmake/FindPkgLibraries.cmake

```diff
@@ -14,13 +14,13 @@ function(FIND_PKGLIBRARIES prefix package)
 	foreach(LIBRARY ${${prefix}_LIBRARIES})
 		find_library(${LIBRARY}_RESOLVED ${LIBRARY} PATHS ${${prefix}_LIBRARY_DIRS})
 		if(${${LIBRARY}_RESOLVED} STREQUAL "${LIBRARY}_RESOLVED-NOTFOUND")
-			message(FATAL_ERROR "could not resolve ${LIBRARY}")
+			message(WARNING "could not resolve ${LIBRARY}")
```

Comment:
This is my only question - why are we changing this from a fatal to a warning? What's the operational change that this makes? If - for example - I asked for libssh2 on the command line and we couldn't find it, that should be a hard fail, shouldn't it?

Reply:
Perhaps another solution would be to skip-list known windows sdk libraries.